CombinatorialGames/Game/Specific/Poset.lean

```diff
@@ -72,10 +72,9 @@ theorem posetRel_univ_of_posetRel_top_compl {α : Type*} [PartialOrder α] [Orde
 theorem wellFounded_posetRel [WellQuasiOrderedLE α] : @WellFounded (Set α) (· ≺ ·) := by
   rw [WellFounded.wellFounded_iff_no_descending_seq]
   refine ⟨fun ⟨f, hf⟩ ↦ ?_⟩
-  have hf' := hf -- Is there a way to make `choose` not delete my hypothesis?
-  choose g hg using hf
+  choose g hg using id hf
```

Comment:
[Not against this, but how?]

Reply:
I learned about this here: https://github.com/leanprover-community/mathlib4/pull/27225